plugins/toolkit/jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectionProvider.kt

```diff
@@ -436,7 +436,8 @@ class CawsConnectionProvider : GatewayConnectionProvider {
                 }
 
                 is GitSettings.CawsOwnedRepoSettings,
-                is GitSettings.NoRepo -> {
+                is GitSettings.NoRepo,
```

Comment:
is this the correct syntax, do we want the comma here?

Reply:
yes it is multi line so it makes sense